package.json

```diff
@@ -47,6 +47,7 @@
     "clean": "rimraf ./dist",
     "prebuild": "npm run clean",
     "build": "tsup",
+    "prepare": "npm run build",
```

Comment:
I don't understand this addition.. you want to build the addon right after its preparation? It's not very useful, is it ?

Reply:
It's just npm build-process-related, to let me do this while I await your merging the PR:

https://github.com/CivicTechTO/polis-storybook/blob/990366525a0d6dc2b7edc6ebf0d2018b6eed6885/package.json#L33

I would say it's a good practice when maintaining a package, so that everyone can have dev workflows that can more easily use pending PRs before you have time to review, approve, and cut a new release into npm registry :)

Reply:
I'm happy to put it into its own PR (I cut corners a bit by including it here...!)

Reply:
TLDR: this allows 'npm install' on the above referenced repo to work. Otherwise it won't

Reply:
Okay, I understand the need behind the addition of this command. No problem for me, I'm fine with it!